.travis.yml

```diff
@@ -20,6 +20,8 @@ deploy:
   provider: releases
   api_key: $DEPLOY_TOKEN_SECURE
   file: dist/roave-backward-compatibility-check.phar
+  # do not skip cleanup: if the repo is cleaned up when deploying, then the phar is gone as well.
+  skip_cleanup: true
```

Comment:
Hmm, I read that this is deprecated and does nothing, the default is to skip cleanup now.

Reply:
It was still affecting the build - in fact, 4.4.0 has a phar, and 4.3.0 doesn't :-\

Reply:
Humm I must've been mistaken then! Sorry, I removed it because Travis config checker was complaining about it heh. Maybe different build setup or something idk.

Reply:
Not a problem: iz fixed nao :)